### ISSUE TITLE
TIFF file format is not supported for Android

### ISSUE THREAD

**Issue by tragicmj** [DylanVann/react-native-fast-image#892]
**Describe the bug**
I am working on a market place app, where users can also chat. TIFF format images are not working  for android, it works for iOS.

**Dependency versions**

- React Native version: 0.66.1
- React version: 17.0.2
- React Native Fast Image version: 8.5.11

**Comment by dimaag-prashanth** [comment]
Hi
I'm working on same use case 
did you find any solution to render TIFF file.

**Comment by tragicmj** [comment]
No. Tiff files are by default not supported in android as far as I have checked.
